src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationService.java

```diff
@@ -47,34 +54,78 @@ public void createAndSendNotification(NotificationWrapper wrapper) {
             Notification notification = factory.create(wrapper);
 
             if (wrapper.getNotificationType().isAllowOutOfHours() || !outOfHoursCalculator.isItOutOfHours()) {
-                if (appellantSubscription.isEmailSubscribed() && notification.isEmail() && notification.getEmailTemplate() != null) {
-                    NotificationHandler.SendNotification sendNotification = () ->
-                            notificationSender.sendEmail(
-                                    notification.getEmailTemplate(),
-                                    notification.getEmail(),
-                                    notification.getPlaceholders(),
-                                    notification.getReference(),
-                                    wrapper.getCaseId()
-                            );
-                    notificationHandler.sendNotification(wrapper, notification.getEmailTemplate(), "Email", sendNotification);
-                }
-                if (appellantSubscription.isSmsSubscribed() && notification.isSms() && notification.getSmsTemplate() != null) {
-                    NotificationHandler.SendNotification sendNotification = () ->
-                            notificationSender.sendSms(
-                                    notification.getSmsTemplate(),
-                                    notification.getMobile(),
-                                    notification.getPlaceholders(),
-                                    notification.getReference(),
-                                    notification.getSmsSenderTemplate(),
-                                    wrapper.getCaseId()
-                            );
-                    notificationHandler.sendNotification(wrapper, notification.getSmsTemplate(), "SMS", sendNotification);
-                }
+                sendEmailSmsNotification(wrapper, appellantSubscription, notification);
+                processOldSubscriptionNotifications(wrapper, notification);
             } else {
                 notificationHandler.scheduleNotification(wrapper);
             }
 
             reminderService.createReminders(wrapper);
         }
     }
+
+    private void processOldSubscriptionNotifications(NotificationWrapper wrapper, Notification notification) {
+        if (wrapper.getNotificationType() == NotificationEventType.SUBSCRIPTION_UPDATED_NOTIFICATION
+                && wrapper.getHearingType() == AppealHearingType.PAPER) {
```

Comment:
Is this only required for paper cases? Why not both types?

Reply:
As of now its only for PAPER. Will check with BA for other type.